src/System.Runtime.Extensions/tests/System/Runtime/Versioning/VersioningHelperTests.cs

```diff
@@ -10,7 +10,8 @@ namespace System.Runtime.Versioning.Tests
     public static class VersioningHelperTests
     {
         [Fact]
-        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "r3 suffix is hardcoded in .Net Core and return code is not documented")]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework | TargetFrameworkMonikers.UapAot, 
+            "r3 suffix is hardcoded in .Net Core and return code is not documented")]
```

Comment:
We're skipping netfx because it's not our purpose to be a netfx test suite. It is our purpose to be a uapaot test suite, although it's fine here for the return value to be different to Core. Even if it's not a stable return value, we should check we get one.